gsy_framework/schema/avro_schemas/launch_simulation_scenario.json

```diff
@@ -0,0 +1,341 @@
+{
+  "name": "ScenarioArea",
+  "type": "record",
+  "namespace": "my.types",
+  "fields": [
+    {"name": "name", "type": "string"},
+    {"name": "type", "type": ["null", "string"], "default":  "undefined"},
+    {"name": "uuid", "type": "string"},
+    {"name": "configuration_uuid", "type": ["null", "string"], "default":  "undefined"},
+    {"name": "collaboration_uuid", "type": ["null", "string"], "default":  "undefined"},
+    {"name": "tags", "type": [
+      "null", {
+        "type": "array",
+        "items":  "string"
+      }
+    ]},
+    {"name": "address", "type": ["null", "string"]},
+    {"name": "libraryUUID", "type": ["null", "string"]},
+    {"name": "feed_in_tariff", "type": ["null", "float"]},
+    {"name": "taxes_surcharges", "type": ["null", "float"]},
+    {"name": "fit_area_boundary", "type": ["null", "boolean"]},
+    {"name": "fixed_monthly_fee", "type": ["null", "float"]},
+    {"name": "grid_fee_constant", "type": ["null", "float"]},
+    {"name": "market_maker_rate", "type": ["null", "float"]},
+    {"name": "geo_tag_location", "type": [
+      "null", {
+        "type": "array",
+        "items":  "float"
+      }
+    ]},
+    {"name": "target_market_kpi", "type": ["null", "int"]},
+    {"name": "export_capacity_kVA", "type": ["null", "float"]},
+    {"name": "import_capacity_kVA", "type": ["null", "float"]},
+    {"name": "grid_fee_percentage", "type": ["null", "float"]},
+    {"name": "coefficient_percentage", "type": ["null", "float"]},
+    {"name": "marketplace_monthly_fee", "type": ["null", "float"]},
+    {"name": "allow_external_connection", "type": ["null", "boolean"]},
+    {"name": "baseline_peak_energy_export_kWh", "type": ["null", "float"]},
+    {"name": "baseline_peak_energy_import_kWh", "type": ["null", "float"]},
+    {"name": "children", "type": [
```

Comment:
I have to admit, that I did not review all lines in detail here (if you want me to, please let me know), but I do not see an entry for the HeatPump. Shall we add it in the frame of https://github.com/gridsingularity/gsy-framework/pull/427 ?

Reply:
Darn, I had a discussion with @mrsaemir about this but I unfortunately forgot to add it. Let me quickly do it now, thanks for noticing !

Reply:
We also had a discussion about modifying the log handler when running tests and report ERROR logs as test failures. I just remembered it now!